deploy/template.yaml

```diff
@@ -54,12 +54,9 @@ Conditions:
     - !Equals [!Ref Environment, staging]
     - !Equals [!Ref Environment, integration]
     - !Equals [!Ref Environment, production]
+
```

Comment:
```suggestion
```
Not related to this PR but the `IsNotDevelopment` condition should just be the one liner `!Not [!Equals [!Ref Environment, dev]]` - this would improve readability